features/conversation/conversation.tsx

```diff
@@ -296,7 +295,15 @@ const Messages = memo(function Messages(props: {
             isLatestMessageSentByCurrentUser={
               latestMessageIdByCurrentUser === messageId
             }
-            animateEntering={index === 0}
+            animateEntering={
+              index === 0 &&
+              // Need this because otherwise because our optimistic updates, we first create a dummy message with a random id
```

Comment:
we shouldn't create the optimistic message with random id but hash the conversationId, userid and message content and maybe tag it with the prefix `optimistic:` or some other way to deliniate between the two.